### PR TITLE
Login page and simple dashboard with two links

### PR DESCRIPTION
**Pull request by slurmsmackenzie99**
Simple dashboard with link to current farm and add new

### CODE REVIEW

File: pages/account/loginLayout.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+            iconPosition='left'
+            placeholder='Password'
+            type='password'
+          />
+          <Button color='teal' fluid size='large'>
+            Login
+          </Button>
+        </Segment>
+      </Form>
+      <Message>
```
Comment (Albert-Hadacek) [inline pages/account/loginLayout.js]: This section can be removed as we are not gonna use it :)

File: pages/index.js [hunk 2]
```diff
@@ -1,68 +1,25 @@
...
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
-        </div>
+        <LoginForm></LoginForm>
```
Comment (Albert-Hadacek) [inline pages/index.js]: As you don't pass any content (children) to the Login form component, you can simply use this cleaner `<LoginForm/>` form. Further info: https://react-cn.github.io/react/tips/self-closing-tag.html

File: pages/account/loginLayout.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+import React from 'react'
```
Comment (Albert-Hadacek) [inline pages/account/loginLayout.js]: Let's keep components outside of the pages folder, it is not the best practice as Next.js uses the folder structure for routing. Just create components folder in the root :)

**Review by Albert-Hadacek — CHANGES_REQUESTED** [review]

**Comment by slurmsmackenzie99** [comment]
Can't see why deployment failed
https://vercel.com/deployments/wind-turbines-dvvtb2zz2-albert-hadacek.vercel.app?redirect=1 throws 404

**Pull request merged**